spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestExpireSnapshotsAction.java

```diff
@@ -967,7 +990,77 @@ public void testExpireOlderThanWithRollbackAndMergedManifests() {
             FILE_B.path()), // added, but rolled back
         deletedFiles);
 
-    checkExpirationResults(1, 1, 1, result);
+    checkExpirationResults(1, 0, 0, 1, 1, result);
+  }
+
+  @Test
+  public void testExpireOlderThanWithDeleteFile() {
+    table.updateProperties()
+        .set(TableProperties.FORMAT_VERSION, "2")
+        .set(TableProperties.MANIFEST_MERGE_ENABLED, "false")
```

Comment:
[Question] Any specific reason to disable the manifest merge?

Reply:
Yea as this test checks the exact list of files removed for ExpireSnapshots, I thought it's simpler to get the exact list if we don't have to deal with manifests auto-merging after some time, as then, I just need to get manifests from the latest snapshot.  

Though in this case, I tried and it seems not necessary as the auto-merge does not kick in (default is set high).  I was thinking to keep it in case the algorithm changes and manifests start merging, failing this test, but I am open if it's better to remove it.

Reply:
I see, makes sense to me.

Reply:
I see, makes sense to me to keep the manifest merge as false here.